src/metpy/calc/thermo.py

```diff
@@ -973,12 +971,6 @@ def _parcel_profile_helper(pressure, temperature, dewpoint,
     temp_upper = moist_lapse(unique[::-1], temp_lower[-1]).to(temp_lower.units)
     temp_upper = temp_upper[::-1][indices]
 
-    # Do the virtual temperature correction for the parcel above the LCL
-    if assumptions.use_virtual_temperature:
-
-        # Calculate the mixing ratio and virtual temperature
-        rv_upper = mixing_ratio(saturation_vapor_pressure(temp_lcl), press_upper)
-        temp_upper = virtual_temperature(temp_upper, rv_upper).to(temp_lower.units)
 
     # Return profile pieces
     return (press_lower[:-1], press_lcl, press_upper[1:],
```

Comment:
Also, after the above changes, should make sure that the temp_lcl here is actually the *virtual* temperature at the *original* LCL to be consistent with the pieces of the profile above and below.

Reply:
Shouldn't this be temp_upper, like you mentioned before?

Reply:
Yes.  I meant the temp_lcl in the return statement. The part that is in the red deleted block should be temp_upper....

Reply:
For whatever reason, when you click on the piece of code to comment on (in this case, I clicked on line 976 where the return statement is), it doesn't show the precise line to everyone else, so that led to some confusion in this case....